lua/dyn_cfg.lua

```diff
@@ -1,18 +1,19 @@
-return function (gk_conf, gt_conf, numa_table)
+return function (net_conf, gk_conf, gt_conf, numa_table)
 
 	--
 	-- Configure the variables below for the Dynamic Config block.
 	--
 
 	-- These parameters should likely be initially changed.
 	local log_level = staticlib.c.RTE_LOG_DEBUG
+	local mode = staticlib.c.S_IRWXU
```

Comment:
Let's move `mode` to the end of the block `-- These variables are unlikely to need to be changed.`